My updated branch

Masroor0007 opened the pull request:
### Fixes #
Added the Hide docs button


### Changes proposed in this pull request:
The hide docs button to hide it

imolorhe commented on packages/altair-electron/src/app/touchbar.ts:
```diff
@@ -33,10 +40,39 @@ export class TouchbarManager {
+    this.hideDocsButton = new TouchBarButton({
+      label: 'Hide Docs',
+      click: () => {
+        this.actionManager.hideDocs();
```
There is no `hideDocs` action in the action manager. That needs to be added. This will not work.

Masroor0007 replied:
sure 
i have worked on it and added the functionality to it
by adding a new branch.

Masroor0007 commented:
I have just added a new branch to the repo
called (My_Latest_branch) in which i added the functionality to the
HideDocs button

On Fri, Apr 7, 2023 at 3:56 AM Samuel ***@***.***> wrote:

> ***@***.**** requested changes on this pull request.
> ------------------------------
>
> In packages/altair-electron/src/app/touchbar.ts
> <https://github.com/altair-graphql/altair/pull/2166#discussion_r1160306199>
> :
>
> >        ],
>      });
>
>      return touchBar;
>    }
> +
> +  private removeShowDocsButton() {
> +    const { items } = this.showDocsButton.parent;
> +    const index = items.indexOf(this.showDocsButton);
> +    if (index !== -1) {
> +      items.splice(index, 1);
> +    }
> +  }
> +
> +  private addHideDocsButton() {
> +    this.hideDocsButton = new TouchBarButton({
> +      label: 'Hide Docs',
> +      click: () => {
> +        this.actionManager.hideDocs();
>
> There is no hideDocs action in the action manager. That needs to be
> added. This will not work.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/altair-graphql/altair/pull/2166#pullrequestreview-1375646846>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AYO6DOG6LHQWTULDOMN3G73W747IPANCNFSM6AAAAAAWUSPQ7I>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>

imolorhe commented:
Why are you creating new PRs everytime? You can add your commit to your already existing branch. Please read on [how to contribute to a project](https://git-scm.com/book/en/v2/GitHub-Contributing-to-a-Project).
